app/src/test/java/pl/droidsonroids/droidsmap/OfficeUiFeatureViewTest.kt

```diff
@@ -1,29 +1,38 @@
 package pl.droidsonroids.droidsmap
 
 import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.verify
 import org.junit.Before
+import org.junit.Test
 import pl.droidsonroids.droidsmap.base.MapActivityWrapper
-import pl.droidsonroids.droidsmap.feature.office.mvp.OfficePresenter
+import pl.droidsonroids.droidsmap.feature.office.mvp.OfficePresenterContract
 import pl.droidsonroids.droidsmap.feature.office.ui.OfficeUiFeatureView
 
 class OfficeUiFeatureViewTest {
 
     lateinit var activityWrapperMock: MapActivityWrapper
     lateinit var uiFeature: OfficeUiFeatureView
-    lateinit var presenterMock: OfficePresenter
+    lateinit var presenterContractMock: OfficePresenterContract
 
     @Before
     fun setUp() {
```

Comment:
Jednak wolę pozostać tu przy aktualnej implementacji, PreseterContract jest klasą abstrakcyjną i nie można zrobić spy, @Mock na uiFeature wymaga zmiany na klasę open, ostatecznie ląduję z częścią inicjalizacji zrobioną za pomocą @Mock, a częścią w @Before. Ponadto konstrukcja z użyciem @Before jest już we wszystkich testach